query_optimizer/tests/execution_generator/CMakeLists.txt

```diff
@@ -17,39 +17,39 @@ add_test(quickstep_queryoptimizer_tests_executiongenerator_create
          "../quickstep_queryoptimizer_tests_ExecutionGeneratorTest"
          "${CMAKE_CURRENT_SOURCE_DIR}/Create.test"
          "${CMAKE_CURRENT_BINARY_DIR}/Create.test"
-         "${CMAKE_CURRENT_BINARY_DIR}/")
+         "${CMAKE_CURRENT_BINARY_DIR}/Create/")
```

Comment:
This is a fix to another bug that was discovered during the testing of the concurrency bug.
@cramja Perhaps just reset it here and let me quickly send a separate PR to address this.